android/BUILDING.md

```diff
@@ -0,0 +1,24 @@
+# Build OpenThread Commissioner App
+
+This documents describes instructions to build the OT Commissioner (Android) App.
+
+## Prerequisites
+
+- macOS or Linux computer.
+- Install latest Android Studio from [here](https://developer.android.com/studio).
+
+## Build shared library
+
+The Commissioner Android App is built atop of the native OT Commissioner library. There is a `build-commissioner-libs.sh` script in current directory to help building the required libraries:
```

Comment:
The Commissioner Android App is built on top of the native OT Commissioner library.  Run the `build-commissioner-libs.sh` script in this directory to build the required libraries: